### DISCUSSION TITLE
Youtube/Google accout needed?

### DISCUSSION THREAD

**Discussion by notDavid** [Anarios/return-youtube-dislike#521] [Q&A] [answered]
Hi there,

so does this extension work (in Firefox, macOS) without a Youtube/Google account, or do you need to have an account and be signed in?

Thank you.

**Comment by PickleNik** [comment, marked as answer]
No, you do not need to be logged in for the extension to work.

**Reply by notDavid** [reply]
Strange, i can see the votes, but i can't seem to vote for some reason?

And this is on a clean Firefox install/profile... see the screencast i attached.
So does it only show the numbers, or should i be able to downvote (without logging in to Youtube.) ?

Thank you!

https://user-images.githubusercontent.com/1530401/161904740-11fb0959-0755-4c84-8da3-e12e207b56f1.mp4

**Reply by PickleNik** [reply]
You need to be logged in to vote in YouTube itself. We don't control the likes and it would be unfair to only allow dislikes.

**Reply by notDavid** [reply]
Thank you for confirming... that makes sense.
